main.py

```diff
@@ -0,0 +1,24 @@
+from pymystem3 import Mystem
```

Comment:
please remove files not related to labs assignment

Reply:
fixed